### PR TITLE
updated orchestrator docs

### PR DESCRIPTION
**Pull request by scheyal**
orchestrator docs

### CODE REVIEW

File: experimental/orchestrator/README.md [hunk 1]
```diff
@@ -1,37 +1,55 @@
...
+**Intent Recognizer**: You can use Orchestrator as an intent recognizer with [Adaptive dialogs][6]. Using the same approach as in the dispatch scenario above in order to route to responses within your bot logic.
+
+**Entity Extraction** is not supported yet.  It is on the planning roadmap to add entity extraction in the future.
+
+## Authoring Experience
+
+Orchestrator can be used in different development environments:
+
+* **Code First**: Orchestrator can be integrated into your code project by replacing luis and/or qnamaker for language understanding.  See [Runtime Integration][# Runtime Integration] section for more.
+* [Bot Framework Composer][19]: Orchestrator can be selected as a recognizer within Bot Framework Composer. At this point there are limitations to using Orchstrator in Composer primarily around importing of existing models and tuning up recognition performance (* currently available only if the feature flag is enabled with Composer).
```
Comment (tsuwandy) [inline experimental/orchestrator/README.md:23]: Orchstrator->Orchestrator

File: experimental/orchestrator/README.md [hunk 2]
```diff
@@ -1,37 +1,55 @@
...
-Here's the end to end authoring experience.
+**Intent Recognizer**: You can use Orchestrator as an intent recognizer with [Adaptive dialogs][6]. Using the same approach as in the dispatch scenario above in order to route to responses within your bot logic.
+
+**Entity Extraction** is not supported yet.  It is on the planning roadmap to add entity extraction in the future.
+
+## Authoring Experience
+
+Orchestrator can be used in different development environments:
+
+* **Code First**: Orchestrator can be integrated into your code project by replacing luis and/or qnamaker for language understanding.  See [Runtime Integration][# Runtime Integration] section for more.
```
Comment (tsuwandy) [inline experimental/orchestrator/README.md:22]: Capitalize luis->LUIS 

qnamaker -> QnA Maker

File: experimental/orchestrator/README.md [hunk 3]
```diff
@@ -1,37 +1,55 @@
...
+* [Bot Framework Composer][19]: Orchestrator can be selected as a recognizer within Bot Framework Composer. At this point there are limitations to using Orchstrator in Composer primarily around importing of existing models and tuning up recognition performance (* currently available only if the feature flag is enabled with Composer).
+
+Thus, use of [BF command line tool][7] to prepare and optimize the model for your domain is required in most, if note all, use cases.   To illustrate the workflow, here is a sample the end to end authoring experience:
```
Comment (tsuwandy) [inline experimental/orchestrator/README.md:25]: if note all -> if not all

**Review by tsuwandy — APPROVED** [review]

File: experimental/orchestrator/README.md [hunk 4]
```diff
@@ -1,37 +1,55 @@
...
+
+* Pre-requisite: Install [BF CLI Orchestrator plugin][11] first.
 
-* Pre-requisite: Install [BF CLI Orchestrator plugin][11] first. 
+1. Author Intent-utterances example based .lu definition referred to as a *label file* using the Language Understanding practices as described in [Language Understanding][2] for dispatch (e.g. author .lu file or within the [Composer][3] GUI experience).
+   * Alternatively, [export][8] your LUIS application and [convert][9] to .lu format or [export][10] your QnA Maker KB to .qna format 
+2. Download Natural Language Representation ([NLR][20]) base Model (will be referred to as the *basemodel*) using the `bf orchestrator:basemodel:get` command. 
+   * See `bf orchestrator:basemodel:list` for alternate models. You may need to experiment with the different models to find which performs best for your language domain.
+3. Combine the label file .lu from (1) with the base model from (2) to create a *snapshot* file with a .blu extension.
+   * Use [`bf orchestrator:create`][16] to create just the .blu snapshot file.
```
Comment (tsuwandy) [inline experimental/orchestrator/README.md:40]: This is dispatch scenario mostly, one snapshot file for all Lu/json/qna tsv files.

File: experimental/orchestrator/README.md [hunk 5]
```diff
@@ -1,37 +1,55 @@
...
+   * Use [`bf orchestrator:build`][17] to **TBD**
```
Comment (tsuwandy) [inline experimental/orchestrator/README.md:41]: Composer adaptive/dialog scenario, one blu file for each lu file

**Review by tsuwandy — APPROVED** [review]

**Review by CezaryMarcjan — APPROVED** [review]

**Review by EricDahlvang — APPROVED** [review]
The [Readme.md](https://github.com/microsoft/BotBuilder-Samples/blob/8be04e556d573449dabfdf480b70d208fa0979ff/experimental/orchestrator/README.md) has duplicate code for JS: and CS: samples

File: experimental/orchestrator/README.md [hunk 6]
```diff
@@ -1,37 +1,55 @@
...
-Here's the end to end authoring experience.
+**Intent Recognizer**: You can use Orchestrator as an intent recognizer with [Adaptive dialogs][6]. Using the same approach as in the dispatch scenario above in order to route to responses within your bot logic.
+
+**Entity Extraction** is not supported yet.  It is on the planning roadmap to add entity extraction in the future.
+
+## Authoring Experience
+
+Orchestrator can be used in different development environments:
+
+* **Code First**: Orchestrator can be integrated into your code project by replacing LUIS and/or QnA Maker for language understanding.  See [Runtime Integration][# Runtime Integration] section for more.
```
Comment (scheyal) [inline experimental/orchestrator/README.md:22]: Need to rephrase this one.

**Pull request merged**